mp/conwebsock.py

```diff
@@ -54,11 +55,13 @@ def __init__(self, ip, password):
 
         self.timeout = 5.0
 
-        if b'Password:' in self.read(256, blocking=False):
+        if b'Password:' in self.read(10, blocking=False):
```

Comment:
This line confuses me, it's 10 in master as well. What is Github diffing against? I don't understand.

Reply:
https://github.com/wendlers/mpfshell/blob/master/mp/conwebsock.py#L58

Reply:
Maybe my version is out of date